### ISSUE TITLE
LaTeX title page vertical spacing of items 

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#117]
I have, in `<titlepage>`:

```
        <author>
            <personname>Steve Simonds</personname>
            <department>Department of Mathematics</department>
            <institution>Portland Community College</institution>
            <email>ssimonds@pcc.edu</email>
        </author>

        <editor>
            <personname>Alex Jordan</personname>
            <department>Department of Mathematics</department>
            <institution>Portland Community College</institution>
            <email>alex.jordan@pcc.edu</email>
        </editor>
```

In the pdflatex output, the line with "Steve Simonds" looks nice, with "Portland Community College" beneath it.

But the line with "Alex Jordan, Editor" has too much vertical space before "Portland Community College". It should be consistent with how `<author>` was treated.

Also, my title, inside `<book>`, is `<title>Portland Community College MTH 251 Lab Manual</title>`. This is so long that it moves to a second line. The second line appears to me to be a bit too crowded with the first. This may just be to my eyes though.

**Comment by rbeezer** [comment]
I'll look at spacing after this week's massive work on foundational code (which 
will make many feature requests possible cleanly).

For your long title, try putting a "< br / >" in wherever you think it is 
natural.  This should behave for LaTeX and HTML title page, but maybe it should 
get killed in HTML banner.  Or you can use a "subtitle" on "book" (only).

Rob

**Comment by Alex-Jordan** [comment]
Using `<br />` in the title, then in the half-title page, the second line is not centered. Something about \centering doesn't care for the line break. I locally (in my customization layer) changed it to use \begin{center}...\end{center}.

**Comment by rbeezer** [comment]
Seen it.  \newline and \\ behave differently in different places.  Do you mind making a Github issue?  Should be easy fix.

**Comment by rbeezer** [comment]
The center environment (`begin/end{center}`) adds vertical space before and after.  So I have been avoiding it and using `\centering`.  Which needs a concluding `\\` on the last line to avoid what you are seeing.

Should now be fixed at:  a280d37c
